Tella/Data/Database/Vault/VaultDataBase.swift

```diff
@@ -146,7 +163,39 @@ class VaultDatabase : DataBase, VaultDataBaseProtocol {
             return nil
         }
     }
+
+    func getNonUpdatedVaultFiles() -> [VaultFileDB] {
+        do {
+            let vaultFilesDict = try statementBuilder.getSelectQuery(tableName: VaultD.tVaultFile,
+                                                                     equalCondition:[KeyValue(key: VaultD.cEncryptionUpdated, value: false)])
+            
+            let vaultFiles = vaultFilesDict.compactMap{VaultFileDB.init(dictionnary: $0)}
```

Comment:
maybe let's introduce  a private method `createVaultFile` to handle the creation of VaultFileDB objects? -> allowing for better separation of concerns?

Reply:
or maybe an array extension like this:

```
extension Array where Element == Dictionary<String,Any>{
      func getVaultFiles() -> [VaultFileDB] {
        return self.compactMap{VaultFileDB.init(dictionnary: $0)}
    }
}
```